install/first_google_boot.sh

```diff
@@ -215,21 +219,42 @@ function extract_spinnaker_kube_credentials() {
     kube_zone=$MY_ZONE
   fi
 
+  if clear_metadata_to_file "kube_config" $config_path; then
```

Comment:
Right now I only enable Kubernetes if the credentials are successfully extracted to avoid producing noise in clouddriver when the culprit/cause is this script. But I don't feel strongly about this approach over yours.